en/editor/publish/build-panel.md

```diff
@@ -25,7 +25,6 @@ Make sure the content in the **Scene** panel is saved before you build. If it is
 ![build-task](build-panel/build-task.jpg)
 
 > **Note**: __Cocos Creator__ uses the **Platform** name as the name for the release package generated after the build, and overwrites the original package with each build.
-
```

Comment:
This blank line cannot be removed, otherwise there will be no segmentation effect：

![image](https://user-images.githubusercontent.com/19702899/116197206-0575ac00-a767-11eb-84aa-403881940b59.png)

and multiple "notes" should be written in the following format:

> **Notes**:
>
> 1. Cocos Creator uses the **Platform** name as the name for the release package generated after the build, and overwrites the original package with each build.
> 2. Cocos Creator uses the **Build Task Name** as the name of the release package that is generated after the build, and a new release package is generated with each build. If you want to overwrite the original release package, you can manually modify the **Build Task Name** to match the original release package name.